docs/Kube-API-examples.md

```diff
@@ -56,7 +58,7 @@ In case that the Bare Metal Operator is installed, the Baremetal Agent Controlle
     - Role: master/worker
     - Hostname (optional for user to set)
     - MachineConfigPool (optional for user to set)
-- Reconcile the BareMetalHost hardware details by copying the Agent's inventory data to the BMH's hardwaredetails annotation.
+- Reconcile the BareMetalHost hardware details by copying the Agent's inventory data to the BMH's hardware details annotation.
```

Comment:
It is called `"inspect.metal3.io/hardwaredetails"`, so maybe add quotes around it?

Reply:
looks okay now?